CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # Changelog
 
+## v0.12.0
+
+### Device support
+- Added support for non-Snapdragon powered devices. Most chips on the market are supported now, 
+including Exynos by Samsung, Kirin by Huawei, Mediatek, etc.
+
+### Vision
+- Added `Japan` country
```

Comment:
I guess `- Added Japan sign classificator` is more meaningful WDYT

Reply:
It's not only sign classifier - we detect the country and return in via `onCountryUpdated` callback as well

Reply:
I would also mention why we actually added such country (because of sign classificator) but up to you.

Reply:
Modified similar to iOS, ptal

Reply:
Nice.